daraja/src/commonMain/kotlin/com/vickbt/darajakmp/utils/Utils.kt

```diff
@@ -26,19 +26,20 @@ internal fun Instant.getDarajaTimestamp(): String {
     val currentDateTime = this.toLocalDateTime(TimeZone.currentSystemDefault())
 
     val year = currentDateTime.year
-    val month =
-        if (currentDateTime.monthNumber < 10) "0${currentDateTime.monthNumber}" else currentDateTime.monthNumber
-    val dayOfMonth =
-        if (currentDateTime.dayOfMonth < 10) "0${currentDateTime.dayOfMonth}" else currentDateTime.dayOfMonth
-    val hour = if (currentDateTime.hour < 10) "0${currentDateTime.hour}" else currentDateTime.hour
-    val minutes =
-        if (currentDateTime.minute < 10) "0${currentDateTime.minute}" else currentDateTime.minute
-    val seconds =
-        if (currentDateTime.second < 10) "0${currentDateTime.second}" else currentDateTime.second
+    val month = currentDateTime.monthNumber.asFormattedWithZero()
+    val dayOfMonth = currentDateTime.dayOfMonth.asFormattedWithZero()
+    val hour = currentDateTime.hour.asFormattedWithZero()
+    val minutes = currentDateTime.minute.asFormattedWithZero()
+    val seconds = currentDateTime.second.asFormattedWithZero()
 
     return "$year$month$dayOfMonth$hour$minutes$seconds"
 }
 
+internal fun Int.asFormattedWithZero(): Comparable<*> = when (this < 10) {
```

Comment:
- Add comment.
- Add tests
- Changes need to go through `develop` branch before `main` branch

Reply:
@VictorKabata I have addressed your comments.